src/main/java/org/jenkinsci/main/modules/sshd/PublicKeyAuthenticatorImpl.java

```diff
@@ -34,36 +29,13 @@ public boolean authenticate(String username, PublicKey key, ServerSession sessio
             return false;
         }
 
-        // TODO: use sshKey.has() when we can depend on 1.446 or later
-        if (!sshKey.isAuthorizedKey(getPublicKeySignature(key))) {
-            LOGGER.fine("Key signature didn't match for the user: "+username+" : "+getPublicKeySignature(key));
+        if (!sshKey.has(key)) {
+            LOGGER.fine("Key signature didn't match for the user: "+username);
```

Comment:
Sorry, I missed this comment. We can probably print out the whole public key, but the signature generation relies on Trilead which is what we're wanting to remove, so I'm not keen on still trying to generate a signature for logging purposes.